api-v2/src/plan/entities/plan.entity.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import { CourseWarning, IWarning } from '../../../../frontend/src/models/types';
+import { Schedule } from '../../../../common/types';
+import { Student } from '../../student/entities/student.entity';
+
+@Entity()
+export class Plan {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  name: string;
+
+  @ManyToOne(
+    () => Student,
+    student => student.plans,
+    {
+      cascade: true,
+      onDelete: 'CASCADE',
```

Comment:
yeah that's true. your call! I think it would be acceptable to either cascade, or just have two calls to `delete` in the handler (one for student, one for `delete from plans where student_id = <id>`)

Reply:
either ok